common/scripts/upgrade-imodeljs-native.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+usage() {
+  echo "Creates and pushes a PR branch to upgrade the itwinjs-core master branch to a new version of @bentley/imodeljs-native."
+  echo "Usage: $0 -v VERSION -s SHA"
+  echo "  -v VERSION: The semantic version (e.g., 3.2.7) of the @bentley/imodeljs-native package to which to update."
+  echo "  -s SHA: The SHA hash of the commit on the imodel02 branch to merge into the PR branch."
+}
+
+# Find repo root and verify it's an itwinjs-core repo.
+RepoRoot=`git rev-parse --show-toplevel`
+if [[ $? -ne 0 || ! -f "$RepoRoot/core/backend/package.json" ]]; then
+  >&2 echo "This script must be executed from within itwinjs-core repo."
+  exit 1;
+fi
+
+while getopts "v:s:" options; do
+  case "${options}" in
+    v) AddonVersion=${OPTARG} ;;
+    s) CommitHash=${OPTARG} ;;
+    ?) usage ; exit 1 ;;
+  esac
+done
+
+# Infer version by bumping current patch release and/or infer commit hash by fetching remote tip? Meh, better to require user to be explicit.
```

Comment:
Yea, I think being explicit is needed for now since you'll need to decide between patch and minor bumps.